node/pkg/driver/device_connectivity/device_path_s390x.go

```diff
@@ -0,0 +1,5 @@
+package device_connectivity
+
+// https://cgit.freedesktop.org/systemd/systemd/tree/src/udev/udev-builtin-path_id.c#n613
```

Comment:
Remove the link to c file.
It doesn;t help to understand and also can change or be different depending on os version

Reply:
![image](https://user-images.githubusercontent.com/57443811/75152298-60e67d00-5711-11ea-9d20-a6811aea0273.png)